pkg/controller/velerobackup/velerobackup_controller.go

```diff
@@ -129,6 +146,22 @@ func (r *ReconcileBackup) Reconcile(request reconcile.Request) (reconcile.Result
 		return reconcile.Result{}, err
 	}
 
+	// Check to see how long since the last back was taken (we may need to rate limit)
```

Comment:
I would rather skip over this entire part if `!found` rather than implicitly relying on the time since the default `Time` being greater than the reconcile rate limit duration.